packages/sanity/src/structure/panes/document/documentPanel/header/perspective/DocumentPerspectiveMenu.tsx

```diff
@@ -25,6 +26,11 @@ export function DocumentPerspectiveMenu(): JSX.Element {
     router.navigateIntent('release', {slug})
   }, [router, slug])
 
+  const bundlesMenuButton = useMemo(
+    () => <Button icon={ChevronDownIcon} mode="bleed" padding={2} space={2} />,
```

Comment:
I think this is the same as the ui-components/button, it will just require a `tooltipProps={{content: "See document versions"}}` prop

Reply:
This was to avoid passing a new component through on every render to BundlesMenu

Reply:
Yeah, I meant for the import of the button from `@sanity/ui` instead from `ui-components`

Reply:
- [ ] Use both UI buttons in `DocumentPerspectiveMenu`